host.go

```diff
@@ -46,16 +46,16 @@ func (host *Host) SendRemoteAndRun(job *Job) (err error) {
 	defer host.StopMaster()
 
 	// make cozy
-	err = host.SSH(job, "mkdir -p $HOME/.judo")
-	workdir, err := host.SSHRead(job, "TMPDIR=$HOME/.judo mktemp -d")
+	err = host.SSH(job, "mkdir -p \"$HOME/.judo\"")
```

Comment:
I think this would read much more clearly:

```go
	err = host.SSH(job, `mkdir -p "$HOME/.judo"`)
```

It would've been nice to have a variant of `shquote` that can handle `"`-strings, but it's fine here like this, as these paths are constants.